config.json

```diff
@@ -3,20 +3,34 @@
     "audio/mpeg3",
     "audio/x-mpeg-3",
     "audio/mpeg3",
+    "audio/ogg",
     "image/jpeg",
     "image/gif",
     "image/png",
     "text/plain",
     "video/mp4",
     "video/mpeg",
-    "video/x-mpeg"
+    "video/x-mpeg",
+    "video/x-matroska",
+    "video/webm",
+    "application/x-7z-compressed",
+    "application/pdf",
+    "application/zip",
+    "application/x-sqlite"
+    
+    
   ],
   "maxFilesPerUpload": 3,
   "maxFileSize": 83886080,
   "textPlainExtensions": [
     "css",
     "html",
     "js",
-    "txt"
+    "txt",
+    "JSON",
+    "config",
```

Comment:
add `conf` and `cfg` too maybe, idk if they're as widely used as `ini` and `yaml` tho